spec.md

```diff
@@ -1064,6 +1081,8 @@ In order for a connection to be legal the following conditions must hold:
 3.  Either the flow of the right-hand side expression is source or duplex, or
     the right-hand side expression has a passive type.
 
+4.  The left-hand side and right-hand side types are not property types.
```

Comment:
Are property types allowed in normal FIRRTL aggregates? If so, then this should indicate that neither LHS nor RHS can contain property types. If not, then some language in the property types section would be good.

Reply:
No, property types won't be allowed in aggregates (they will have their own aggregates eventually). I can include some description about this in the property types section.

Reply:
I mentioned that property types "cannot be used in any hardware types, including aggregates"